extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -60,31 +62,19 @@ function createDicomWebApi(dicomWebConfig, UserAuthenticationService) {
     singlepart,
   } = dicomWebConfig;
 
-  const qidoConfig = {
-    url: qidoRoot,
-    staticWado,
-    singlepart,
-    headers: UserAuthenticationService.getAuthorizationHeader(),
-    errorInterceptor: errorHandler.getHTTPErrorHandler(),
-  };
-
-  const wadoConfig = {
-    url: wadoRoot,
-    staticWado,
-    singlepart,
+  const dicomWebClientConfig = {
```

Comment:
Just pass in options directly, or else use 
const wadoConfig = {
...options,
<any custom over-rides}
}